src/main/java/net/minestom/server/item/banner/PatternType.java

```diff
@@ -0,0 +1,75 @@
+package net.minestom.server.item.banner;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum PatternType {
```

Comment:
Can also be moved to the banner meta class. Although I am also not sure if we want to make this an enum?

Reply:
This won't be an enum, was just waiting for the data PR to be merged.